docs/articles/configs/jobs.md

```diff
@@ -118,18 +123,24 @@ Basically, it's a good idea to start with predefined values (e.g. `EnvMode.RyuJi
 
 Note that the job cannot be modified after it's added into config. Trying to set a value on property of the frozen job will throw an `InvalidOperationException`. Use the `Job.Frozen` property to determine if the code properties can be altered.
 
-If you do want to create a new job based on frozen one (all predefined job values are frozen) you can use the `.With()` extension method 
+If you do want to create a new job based on frozen one (all predefined job values are frozen) you can use the `.With()` extension method
+
```

Comment:
ℹ️ In a few cases, my auto-formatter introduced some whitespace that I didn't think you'd hate, so I just left it in instead of undoing it.